src/auxiliary/precompile.jl

```diff
@@ -460,6 +460,9 @@ function _precompile_manual_()
     @assert Base.precompile(Tuple{typeof(trixi_include),String})
   end
 
+  @assert Base.precompile(Tuple{typeof(init_mpi)})
```

Comment:
Why not put this into `@compile_workload`? IMHO that makes it much easier to understand for non experts.

Reply:
I just wanted to compile it, not to run it, since this may have consequences

Reply:
👍